ArduPlane/altitude.cpp

```diff
@@ -643,7 +643,7 @@ void Plane::rangefinder_terrain_correction(float &height)
  */
 void Plane::rangefinder_height_update(void)
 {
-    float distance = rangefinder.distance_cm_orient(ROTATION_PITCH_270)*0.01f;
+    float distance = rangefinder.distance_m_orient(ROTATION_PITCH_270);
```

Comment:
```suggestion
    float distance = rangefinder.distance_orient(ROTATION_PITCH_270);
```

We omit units when they're actually *sensible* :-)

Looking at it, we'd really want to couple that with @IamPete1 's suggestion of getting rid of the "getters" for the parameters.  I think that should probably be done.  I'm torn as to whether it really ought to be done in here.